client/src/components/Ticketing/ticketingmanager/Events/EventForm.tsx

```diff
@@ -169,48 +171,39 @@ const EventForm = ({onSubmit, tickettypes, initialValues}: EventFormProps) => {
       showings: showings,
     };
     console.log(data);
-    if (showings.length === 0) {
-      setErr('Please enter at least one showing.');
+    const name = data.eventname?.trim() === '';
+    const desc = data.eventdescription?.trim() === '';
+    const count = (name?1:0)+(desc?1:0);
+    if (count) {
+      const len = count>1;
+      setErr( `The ${name?'Event Name':''}${len?', and ':''}
+      ${desc?'Event Description':''}
+      ${len?' fields are missing':' field is missing'}`);
```

Comment:
Could you make the variable names used here a little more specific, something like `hasNoName`, `hasNoDesc`, and `missingFieldsCount`.

Reply:
Thanks for simplifying this though, it was a big mess before

Reply:
Updated